src/main/java/com/example/bookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package com.example.bookstore.exception;
+
+public class RegistrationException extends RuntimeException {
+    public RegistrationException(String message) {
+        super(message);
+    }
+}
```

Comment:
According to task requirements we need to handle this exception in GlobalExceptionHandler